editor/store/selectors.js

```diff
@@ -486,6 +486,16 @@ export const getBlocks = createSelector(
 	]
 );
 
+export const getBlocksByUid = createSelector(
```

Comment:
More accurately: `getBlocksByUids`

Also need consensus on capitalization pattern `getBlocksByUids` vs. `getBlocksByUIDs`

Reply:
What's your preference on the capitalization, let's make it a guideline ;)?

Reply:
I agree a guideline would be good. I don't know which I prefer. I think capitalized abbreviations is more the norm, but there's exceptions. It also makes it difficult when a name has two adjacent abbreviations (imagine an `XMLHTTPRequest`).

🤷‍♂️ 

#2511

Reply:
I guess I'd slightly lean toward capitalized abbreviations.